usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -352,7 +352,7 @@ def construct_es_reponse(self, response) -> dict:
             "results": results,
             "page_metadata": {
                 "page": self.pagination["page"],
-                "hasNext": response.hits.total - (self.pagination["page"] - 1) * self.pagination["limit"]
+                "hasNext": response["hits"]["total"]["value"] - (self.pagination["page"] - 1) * self.pagination["limit"]
```

Comment:
Is this a boolean? Usually `hasNext` is true or false

Reply:
`"hasNext": response["hits"]["total"]["value"] - (self.pagination["page"] - 1) * self.pagination["limit"]
                > self.pagination["limit"],` <- the boolean part is on the next line

Reply:
Thanks, sorry. That's what I get for late night PR reviewing.